tests/components/modbus/test_light.py

```diff
@@ -307,21 +347,138 @@ async def test_light_service_turn(
 
 
 @pytest.mark.parametrize(
-    "do_config",
+    (
+        "do_config",
+        "service_data",
+        "expected_brightness",
+        "expected_color_temp",
+    ),
```

Comment:
```suggestion
    ("do_config", "service_data", "expected_calls"),
```

(part of my suggestion for the method bellow, since github wouldn't let me suggest the whole thing in 1 comment)